src/main/java/module-info.java

```diff
@@ -5,6 +5,8 @@
 
 /**
  * Defines the module of this library.
+ *
+ * @since 0.2
```

Comment:
Change to `@version 0.2` because this library would be always this one module only.